Lib/test/test_stat.py

```diff
@@ -173,11 +173,17 @@ def test_link(self):
 
     @unittest.skipUnless(hasattr(os, 'mkfifo'), 'os.mkfifo not available')
     def test_fifo(self):
+        if sys.platform == "vxworks":
+            fifo_path = os.path.join("/fifos/", TESTFN)
+        else:
+            fifo_path = TESTFN
```

Comment:
Instead of calling os.remove(fifo_path) only on VxWorks below, I would prefer to add a more generic code here (similar fix that the one in test_os):

```
self.addCleanup(os_helper.unlink, fifo_path)
```

Just add `from test.support import os_helper` at the top.

Reply:
Modified accordingly.